web/src/info_overlay.js

```diff
@@ -99,7 +106,14 @@ const markdown_help_rows = [
             },
             {open_markdown_syntax: "@**", close_markdown_syntax: "**"},
         ),
-        output_html: '<p><span class="user-mention">@Joe Smith</span></p>',
+        output_html: $t_html(
+            {
+                defaultMessage: "<p><z-mention>@Joe Smith</z-mention></p>",
+            },
+            {
+                "z-mention": (content_html) => `<span class="user-mention">${content_html}</span>`,
+            },
+        ),
```

Comment:
This feels very messy. We end up asking the translator to translate these two version of "Joe Smith" with surrounding syntax for no real purpose. 

Ultimately the ideal way for this function to work is to not need to declare `output_html` at all for these -- then we could just have the string "Joe Smith" in the source and let the Markdown processor process that transparently..

The way to do that would be to replace this logic:

```
     for (const row of markdown_help_rows) {                                                             
        if (row.markdown && !row.output_html) {                                                         
            const message = {raw_content: row.markdown};                                                
            markdown.apply_markdown(message);                                                           
            row.output_html = util.clean_user_content_links(message.content);                           
        }                                                                                               
    }                                                                                                   
```

I think we could instead replace with this calling `markdown.parse` passing a custom version of the `markdown_config.js` function `get_helpers` that just returns stub functions that say, return "Joe Smith" as the name for every user mentioned.

That way, we can delete most of the `output_html` code here by making this info overlay logic actually just process the markdown with an effective user database of "there's a single user called `{translation of Joe Smith}` who can be mentioned".